pkg/test/utils/kubernetes.go

```diff
@@ -967,10 +967,15 @@ func GetArgs(ctx context.Context, cmd harness.Command, namespace string, env map
 	return builtCmd, nil
 }
 
+// To keep interfacer linter happy.
+type Logfer interface {
```

Comment:
why do we need this?   why don't use https://github.com/kudobuilder/kuttl/blob/master/pkg/test/utils/logger.go#L11 ?